domain/modelconfig/validators/validators.go

```diff
@@ -77,6 +78,39 @@ func AuthorizedKeysChange() config.ValidatorFunc {
 	}
 }
 
+// ContainerNetworkingMethodValue checks that the container networking method
+// supplied to model config is a valid value.
+func ContainerNetworkingMethodValue() config.ValidatorFunc {
+	return func(ctx context.Context, cfg, old *config.Config) (*config.Config, error) {
+		switch cfg.ContainerNetworkingMethod() {
+		case "", containermanager.NetworkingMethodLocal.String(),
+			containermanager.NetworkingMethodProvider.String():
+			return cfg, nil
+		}
```

Comment:
@tlm and here too right?
```suggestion
		switch cfg.ContainerNetworkingMethod() {
		case "", "local", "provider":
			return cfg, nil
		}
```